### PR TITLE
feature: import official webview rust binding

### PR DESCRIPTION
**Pull request by wusyong**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] New Binding Issue #___
- [ ] Code style update
- [x] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes. Issue #___
- [x] No

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch and _not_ the `latest` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Comment by nothingismagick** [comment]
# 🎉  🔥 🚀 💯

File: tauri/test/fixture/src-tauri/tauri.conf.json [hunk 1]
```diff
@@ -19,7 +19,9 @@
     },
     "security": {
       "csp": "default-src blob: data: filesystem: ws: http: https: 'unsafe-eval' 'unsafe-inline'"
+    },
+    "inliner": {
+      "active": truea
```
Comment (nklayman) [inline tauri/test/fixture/src-tauri/tauri.conf.json]: `true` not `truea`?

Comment (lucasfernog) [reply tauri/test/fixture/src-tauri/tauri.conf.json]: thanks!

**Comment by wusyong** [comment]
I could only verify the rust part, but it's good to merge imho. After this, I think we could open some refactor issues like `Content` type conversion, `__initialized` bindings and maybe others.

**Comment by lucasfernog** [comment]
I think binding functions (and maybe exposing that API to users) is going to have to wait, because Edge reloads the app and the `window[name]` bindings are gone. (I had to manually set the __INVOKE__ binding every time on JS, but it's just a workaround; talked about it on the webview Discord channel and I might open an issue).

But if you think the Rust part is OK then LGTM

**Review by lucasfernog — APPROVED** [review]

**Comment by lucasfernog** [comment]
@wusyong are you ok with the change file I've added?

**Comment by wusyong** [comment]
Of course! :smile:

**Pull request merged**